pkg/cloud/ovirt/clients/machineservice.go

```diff
@@ -120,6 +120,41 @@ func (is *InstanceService) InstanceCreate(
 		}
 	}
 
+	isAutoPinning := false
+	// TODO: remove the version check when everyone uses engine 4.4.5
+	engineVer := is.getEngineVersion()
+	versionCompareResult, err := versionCompare(engineVer, ovirtsdk.NewVersionBuilder().
+		Major(4).
+		Minor(4).
+		Build_(5).
+		Revision(0).
+		MustBuild())
+	if err != nil {
+		return nil, err
+	}
+	if providerSpec.AutoPinningPolicy != "" && versionCompareResult < 0 {
+		klog.Warning("The engine version %d.%d.%d is not supporting the auto pinning feature. " +
+		"Please update to 4.4.5 or later.", engineVer.MustMajor(), engineVer.MustMinor(), engineVer.MustBuild())
+	} else if providerSpec.AutoPinningPolicy != "" {
```

Comment:
Guess this else if will be removed once the version check will be inside the autoPining condition

Reply:
and left only with `else`